src/main/java/org/opensearch/knn/index/util/KNNEngine.java

```diff
@@ -54,6 +56,10 @@ public static KNNEngine getEngine(String name) {
             return FAISS;
         }
 
+        if (LUCENE.getName().equals(name)) {
+            return LUCENE;
+        }
+
```

Comment:
nit: maybe in next PR we can refactor this to switch/case or lookup in map of <engine_name, engine instance>

Reply:
We could probably use switch.